benchmark/dqn/algorithm.py

```diff
@@ -0,0 +1,88 @@
+#   Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
We would like to add a new folder `torch` in benchmark as we may support tf2.0 as our another benchmark in the near future. Please remove the folder `dqn` into `benchmark/torch/dqn`.